tests/Microsoft.Identity.Test.Integration.net45/HeadlessTests/AuthorityTests.cs

```diff
@@ -88,6 +88,34 @@ public async Task FailedAuthorityValidationTestAsync()
             Assert.AreEqual("invalid_instance", exception.ErrorCode);
         }
 
+        [TestMethod]
+        public async Task AuthorityValidationTestWithFalseValidateAuthorityAsync()
+        {
+            LabResponse labResponse = await LabUserHelper.GetDefaultUserAsync().ConfigureAwait(false);
+            LabUser user = labResponse.User;
+
+            IPublicClientApplication pca = PublicClientApplicationBuilder
+                .Create(labResponse.App.AppId)
+                .WithAuthority("https://bogus.microsoft.com/common", false)
+                .WithTestLogging()
+                .Build();
+
+            Trace.WriteLine("Acquire a token using a not so common authority alias");
+
+            HttpRequestException exception = await AssertException.TaskThrowsAsync<HttpRequestException>(() =>
+                 pca.AcquireTokenByUsernamePassword(
+                    s_scopes,
+                     user.Upn,
+                     new NetworkCredential("", user.GetOrFetchPassword()).SecurePassword)
+                     .ExecuteAsync())
+                .ConfigureAwait(false);
+#if NET_CORE
```

Comment:
There is a difference in exceptions thrown by net standard and net core for unknown host. I have added a condition here. Not sure if this is the best practice.

Reply:
This works fine, thanks Neha.